doc/src/manual/documentation.md

```diff
@@ -453,6 +456,20 @@ end
 
 Adds the docstring `"..."` to types `T1`, `T2`, and `T3`.
 
+```
+"..."
+T1
+
+"..."
+T2
+
+"..."
+T3
+```
+
+Adds the docstring `"..."` to types `T1`, `T2`, and `T3`.
+The previous version is the preferred syntax, however both are equivalent.
```

Comment:
```suggestion
"..."
T3
```

Adds the docstring `"..."` to types `T1`, `T2`, and `T3`.
The previous version is the preferred syntax, however both are equivalent.
```

This feels bad to me, and I'm hesitant to document (and therefore make part of the public API) and also tell people not to do it.

Reply:
But hasn't that ship sailed long ago? I am pretty sure plenty packages use this, for whatever reasons.

Anyway, I am not trying to argue strongly for documenting this.

Reply:
It seems fine to mention. The other syntax is usually preferred since it keeps the documentation with the struct, but there are plenty of case (at least in Base) where the documentation is a separate file of its own